catalog/tests/dags/common/test_resources/image_table_sample.tsv

```diff
@@ -15,6 +15,6 @@
 00004d72-5bde-42c9-bdf0-bcfa2d659076	2020-10-12 18:11:57.791507+00	2020-10-15 08:10:42.421899+00	provider_api	\N	ark:/65665/m3db1a8e6c57b44991ba72f73f11614b1e	https://n2t.net/ark:/65665/398b1d1ca-b7ab-4225-9157-7caaac6c166a	https://ids.si.edu/ids/deliveryService/id/ark:/65665/m3db1a8e6c57b44991ba72f73f11614b1e	https://ids.si.edu/ids/deliveryService/id/ark:/65665/m3db1a8e6c57b44991ba72f73f11614b1e/90	\N	\N	\N	cc0	\N	1.0	K R. Chamberlain	\N	Biotite granite	smithsonian	smithsonian_national_museum_of_natural_history
 000057a3-5ee0-4979-be14-a7674620fd72	2020-10-12 18:11:57.791507+00	2020-10-15 08:10:42.421899+00	provider_api	\N	ark:/65665/m3e3e6c97b62d4406d91e312af11d11047	https://n2t.net/ark:/65665/38e5905d1-8a0f-4783-94a1-9d4c7b7fb10f	https://ids.si.edu/ids/deliveryService/id/ark:/65665/m3e3e6c97b62d4406d91e312af11d11047	https://ids.si.edu/ids/deliveryService/id/ark:/65665/m3e3e6c97b62d4406d91e312af11d11047/90	\N	\N	\N	cc0	\N	1.0	Wilhelm Gehriger	\N	Salvia angulata Benth.	smithsonian	smithsonian_national_museum_of_natural_history
 00005f89-08dd-4631-8e23-f23091cf465b	2020-10-12 18:11:57.791507+00	2020-10-15 08:10:42.421899+00	provider_api	\N	ark:/65665/m3f2b8e24b9bea48758b085187448410ac	https://n2t.net/ark:/65665/349d65504-4339-4627-af1e-e11d601028e6	https://ids.si.edu/ids/deliveryService/id/ark:/65665/m3f2b8e24b9bea48758b085187448410ac	https://ids.si.edu/ids/deliveryService/id/ark:/65665/m3f2b8e24b9bea48758b085187448410ac/90	\N	\N	\N	cc0	\N	1.0	M. G. Harasewych and Linda L. Harasewych	\N	Cerion casablancae	smithsonian	smithsonian_national_museum_of_natural_history
-00006471-c6ba-4f96-8e01-ffd168e07f0e	2020-10-12 13:30:44.317801+00	2020-10-15 08:08:40.252185+00	provider_api	\N	50448966551	https://www.flickr.com/photos/64484371@N03/50448966551	https://live.staticflickr.com/65535/50448966551_13b7497122_b.jpg	https://live.staticflickr.com/65535/50448966551_13b7497122_m.jpg	1024	768	\N	by-sa	\N	2.0	Cristal Montanez Venezuela	https://www.flickr.com/photos/64484371@N03		flickr	flickr
+00006471-c6ba-4f96-8e01-ffd168e07f0e	2020-10-12 13:30:44.317801+00	2020-10-15 08:08:40.252185+00	provider_api	\N	50448966551	https://www.flickr.com/photos/64484371@N03/50448966551	https://live.staticflickr.com/65535/50448966551_13b7497122_b.jpg	https://live.staticflickr.com/65535/50448966551_13b7497122_m.jpg	1024	768	\N	by-sa	\N	2.0	Crystal Montanez Venezuela	https://www.flickr.com/photos/64484371@N03		flickr	flickr
```

Comment:
I'd argue that this file should be excluded from spell-check because we do not control the accuracy of spelling of media titles, tags and descriptions ingested from sources.

Reply:
Also this change seems to be in the name of a media creator, which is wrong.

Reply:
Yes! I missed this one from the ignore files list :+1: